test/services/dynamicsDal.service.test.js

```diff
@@ -57,11 +57,15 @@ lab.beforeEach((done) => {
 
   nock(`https://${config.dynamicsWebApiHost}`)
     .post(`${config.dynamicsWebApiPath}__DYNAMICS_INSERT_QUERY__`)
-    .reply(204)
+    .reply(204, '', {
+      'odata-entityid': `https://${config.dynamicsWebApiHost}${config.dynamicsWebApiPath}contacts(7a8e4354-4f24-e711-80fd-5065f38a1b01)`
+    })
 
   nock(`https://${config.dynamicsWebApiHost}`)
     .patch(`${config.dynamicsWebApiPath}__DYNAMICS_UPDATE_QUERY__`)
-    .reply(204)
+    .reply(204, '', {
+      'odata-entityid': `${config.dynamicsWebApiHost}${config.dynamicsWebApiPath}/contacts(7a8e4354-4f24-e711-80fd-5065f38a1b01)`
```

Comment:
Why doesn't this one include `https://`? Just wondering why the response is different.

Reply:
Good spot. Also indicates I'm not really testing the `update()` method (as in `_commit()` it must be going through the new record branch) so will need to check the test as well.